server/model/user.go

```diff
@@ -60,7 +60,7 @@ type User struct {
 type UserPropPatch struct {
 	// The user prop updated fields
 	// required: false
-	UpdatedFields map[string]string `json:"updatedFields"`
+	UpdatedFields map[string]interface{} `json:"updatedFields"`
```

Comment:
Why do we need this changes, is no way to keep it just `map[string]string`, this kind of things tends to be more fragile. If that is needed, is ok, but if we can solve that using multiple keys instead of a nested object, would result in something more robust IMO.

Reply:
Done.